data/strings/types_strings.txt

```diff
@@ -24979,6 +24988,7 @@
     fa = مرکز خرید
     fi = Tavaratalo
     fr = Grand magasin
+    he = חנות כל-בו גדולה
```

Comment:
Maybe drop the "גדולה" (big), just "חנות כל-בו" is good.

For variety_store, some other term is needed. Maybe חנות מגוון? Direct translation of variety store.

Reply:
already exists "חנות כל-בו"

Reply:
There is no other translation in Hebrew (there is no such thing in Israel either)

Reply:
Want to discuss this on Telegram in Hebrew? Might be easier.